src/api.ts

```diff
@@ -1,7 +1,7 @@
 import { MovieData } from './types/MovieData';
 import { ResponseError } from './types/ReponseError';
 
-const API_URL = 'https://www.omdbapi.com/?apikey=your-key';
+const API_URL = 'https://www.omdbapi.com/?apikey=3ba730e2';
```

Comment:
this should be in .env for sure, but I am not sure if you were prompted to do that 🤔